www/src/views/shared/sidebar.js

```diff
@@ -2,8 +2,10 @@ import React from "react"
 import MdFilterList from "react-icons/lib/md/filter-list"
 import styles from "../shared/styles"
 
-export const SidebarContainer = ({ children }) => (
-  <div css={[styles.sidebarContainer, styles.sticky]}>{children}</div>
+export const SidebarContainer = ({ children, overrideCSS }) => (
+  <div css={[styles.sidebarContainer, styles.sticky, overrideCSS]}>
```

Comment:
I think we clean this up a little bit. Can we try something like the following:

```jsx
export const SidebarContainer = ({ children, className }) => (
  <div className={className} css={[styles.sidebarContainer, styles.sticky]}>{children}</div>
```

(and then see below for more info on how to _use_)

Reply:
(also just for clarification here, the way CSS in JS libraries--like emotion!--work here is that they inject a className, so consuming a class name is a good way to make a component externally stylable with the CSS prop or `styled(SidebarContainer)()`)